<launch>

<!-- declare arg to be passed in -->
 <arg name="workspace_metric_offset_x" default="0.733" />
 <arg name="workspace_metric_offset_y" default="0.455" />
 <arg name="workspace_metric_offset_z" default="0.0" />
 <arg name="visualize" default="False" />
 <arg name="cloud_topic" default="kinect2/hd/points" />
 <arg name="base_frame" default="/yumi_base_link" />
 

<node name="color_segmentation_based_perception_node" pkg="perception_manager" type="color_segmentation_based_perception_node" output="screen">
  <param name="workspace_metric_offset_x" value="$(arg workspace_metric_offset_x)" />
  <param name="workspace_metric_offset_y" value="$(arg workspace_metric_offset_y)" />
  <param name="workspace_metric_offset_z" value="$(arg workspace_metric_offset_z)" />
  <param name="visualize" value="$(arg visualize)" />
   <param name="base_frame" value="$(arg base_frame)" />

</node>

</launch>
